week1/react-exercises/src/App.js

```diff
@@ -0,0 +1,16 @@
+import "./App.css";
+import HobbyList from "./1-hobbies/Hobbies";
+import Guarantee from "./2-guarantee/Guarantee";
+import Counter from "./3-counter/Counter";
+
+function App() {
```

Comment:
Not wrong, it's just better to pick one style and stick with it.

```suggestion
const  App = () => {
```

Reply:
Noted to fix it after creating react app :)